publisher-sdk/src/main/java/com/criteo/publisher/model/RemoteConfigRequest.kt

```diff
@@ -0,0 +1,34 @@
+/*
+ *    Copyright 2020 Criteo
+ *
+ *    Licensed under the Apache License, Version 2.0 (the "License");
+ *    you may not use this file except in compliance with the License.
+ *    You may obtain a copy of the License at
+ *
+ *        http://www.apache.org/licenses/LICENSE-2.0
+ *
+ *    Unless required by applicable law or agreed to in writing, software
+ *    distributed under the License is distributed on an "AS IS" BASIS,
+ *    WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ *    See the License for the specific language governing permissions and
+ *    limitations under the License.
+ */
+package com.criteo.publisher.model
+
+import com.criteo.publisher.annotation.OpenForTesting
+import com.google.gson.annotations.SerializedName
+
+@OpenForTesting
+data class RemoteConfigRequest(
+    @SerializedName("cpId")
+    val criteoPublisherId: String,
+    val bundleId: String,
+    val sdkVersion: String,
+    @SerializedName("rtbProfileId")
+    val profileId: Int,
+    /**
+     * Field used by the remote config to A/B test some configurations.
+     */
+    val deviceId: String?,
+    val deviceOs: String,
```

Comment:
nit: this is not expected to be variable, but a constant (`android`), making it variable could add some interrogation and increase the cognitive load